From 438d95fa2f978d704751e8b509466a80a8c74aef Mon Sep 17 00:00:00 2001
From: Evgueni Souleimanov <esoule@100500.ca>
Date: Wed, 6 Jun 2018 19:13:54 -0400
Subject: [PATCH 1/2] Add missing use statements in uimage handler

Fixes this error:

    $ strip-nondeterminism -T 123123123 Linux-1001.uImage
    strip-nondeterminism: Linux-1001.uImage: Undefined subroutine
    &Archive::Zip::computeCRC32 called at
    /usr/share/perl5/File/StripNondeterminism/handlers/uimage.pm line 50.

Error shows up when trying to use strip-nondeterminism script, but does not
show up in "make test" during build.

Signed-off-by: Evgueni Souleimanov <esoule@100500.ca>
---
 lib/File/StripNondeterminism/handlers/uimage.pm | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/lib/File/StripNondeterminism/handlers/uimage.pm b/lib/File/StripNondeterminism/handlers/uimage.pm
index 1f9ba347e205..1cd4ddf2c6eb 100644
--- a/lib/File/StripNondeterminism/handlers/uimage.pm
+++ b/lib/File/StripNondeterminism/handlers/uimage.pm
@@ -20,6 +20,9 @@ package File::StripNondeterminism::handlers::uimage;
 
 use strict;
 use warnings;
+
+use Archive::Zip ();
+use File::StripNondeterminism;
 use Fcntl q/SEEK_SET/;
 
 # /*
-- 
2.17.0

